### ISSUE TITLE
recording icon not leaving on mute

### ISSUE THREAD

**Issue by maxirozay** [jitsi/lib-jitsi-meet#2127]
## Description
The browser recording icon next to the site title stays after muting the audio.

## Current behavior
Turn off your microphone and the recording icon stays like you were still being recorded.

## Expected Behavior
Turn off your microphone and the icon leaves so you know that you're not being recorded.

## Possible Solution
In [JitsiLocalTrack.js](https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/RTC/JitsiLocalTrack.js#L376) if the track is audio it only set enabled to false `this.track.enabled = !muted;` else it calls `_removeStreamFromConferenceAsMute`.

I tried to change the _setMuted function from `if (this.isAudioTrack()` to `if (this._streamEffect` so if there' an effect it would still use the effect `setMuted` function else it would mute the audio the same way it does for video. It works but then when the unmute part of the function handle only video so I think it could be easily fixed but I don't know if there's a good reason for not really turning the audio off.

## Steps to reproduce
Go to a room on meet.jit.si and turn off your microphone and camera. Look at the recording icon next to the tab title.

# Environment details
All

**Comment by saghul** [comment]
Hey there!

When you mute the mic is not technically closed. No audio is sent to the other side (it's disabled) but it's not closed, for a number of reasons:

- Opening it takes time, so unmuting would be a much more costly operation
- We cannot give user hints such as "are you trying to speak?"

That indicator is technically correct because the mic is still open, even though no data is being transmitted.

**Comment by maxirozay** [comment]
Thank you, I understand I'll close this issue but before I have another question/remark related to this issue.

If my video has an effect I have to call `await video.setEffect()` before `dispose()`. If I don't call and wait `setEffect` the browser icon stays.

The [dispose](https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/RTC/JitsiLocalTrack.js#L600) function call setEffect but maybe it should wait for `setEffect` to finish before calling `stopStream` no ?

**Comment by alimtunc** [comment]
Hello, I've got the same issue today. When you mute the track, browser should said that we don't send data anymore, no? Like jitsi client is working today. Muting mic or disabling video will turn off the camera is recording indicator. But today with jitsi-low-level, the camera indicator is turned on every time.

I've got also an issue when leaving the room with a muted track keeping the camera indicator on.. I think there some troubleshoot with this today. Am I the only one getting these issues?
